notebooks/notebook_template.ipynb

```diff
@@ -43,7 +43,7 @@
         "    </a>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    IS_COLAB = False
</pre><p>Prefer:</p><p> </p><pre class="ql-syntax" spellcheck="false">IS_COLAB = google.colab" in sys.modules
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>updated</p>

Reply:
<p>If kernel restarted, those VARS no longer exist.</p>